sdk/storage/azure-storage-files-shares/test/share_service_client_test.cpp

```diff
@@ -146,6 +146,8 @@ namespace Azure { namespace Storage { namespace Test {
   TEST_F(FileShareServiceClientTest, SetProperties)
   {
     auto properties = *m_fileShareServiceClient->GetProperties();
+    // Has to remove before set, otherwise would return failure.
+    properties.Protocol = Core::Nullable<Files::Shares::ShareProtocolSettings>();
```

Comment:
I don't understand this comment. Can you explain the bug please.

Reply:
Currently, the service is rejecting `ProtocolSettings` to be set for subscriptions that haven't turned this feature on, although this field is returned in the response body.
We are trying to communicate with the service team if this is correct behavior, but in the meantime, this line needs to exist.

Reply:
Then create an issue for this.